lib/OpenQA/Command.pm

```diff
@@ -121,18 +121,20 @@ sub url_for ($self, $path) {
 sub retry_tx ($self, $client, $tx, $retries = undef, $delay = undef) {
     $client->connect_timeout($ENV{MOJO_CONNECT_TIMEOUT} // 30);
     $delay //= $ENV{OPENQA_CLI_RETRY_SLEEP_TIME_S} // 3;
+    my $max_delay = 600;
     $retries //= $ENV{OPENQA_CLI_RETRIES} // 0;
     my $start = time;
-    for (;; --$retries) {
+    do {
         $tx = $client->start($tx);
         my $res_code = $tx->res->code // 0;
         return $self->handle_result($tx) unless $res_code =~ /^(50[23]|0)$/ && $retries > 0;
         my $waited = time - $start;
         print encode('UTF-8',
-"Request failed, hit error $res_code, retrying up to $retries more times after waiting … (delay: $delay; waited ${waited}s)\n"
+            "Request failed, hit error $res_code, retrying up to $retries more times after waiting … (delay: $delay; waited ${waited}s)\n"
         );
         sleep $delay;
-    }
+        $delay = $delay << 1 if $delay > $max_delay;
```

Comment:
I think we don't need a maximum.
If people deliberately specify 10 retries plus a factor it's their choice, and I don't see a big problem with it.

Reply:
Also here you only increase delay if it is bigger then the maximum. Which will never happen unless it is bigger than the maximum from the beginning.
Also we said a normal multiplication would be preferrable.
```
$delay *= $factor;
```

Reply:
And even if we wanted a maximum the condition would probably we wrong. So better just remove it.

Reply:
I have to apologize in advanced. Apparently i dont get it completely. I assume that retry_tx is running in each `openqa-cli` invocation. I try to be responsible of its requests and unlimited ones without think the coincidences sound wrong. Lets say is not a big deal, but to tackle requests which keep failing after retries is not the best design. 
if 5 retries with $delay=3 never reach the max_delay. you need 8 retries. 600sec==10min. Does it really make sense to keep trying or do we need something additional here?

  > my $retry_factor = $ENV{OPENQA_CLI_RETRY_FACTOR} // 1;
  
  Also i think this complicates things. Why to disable it at all? (i think this is open for discussion which i cant win :) But glad to have a productive debate)

Reply:
> And even if we wanted a maximum the condition would probably we wrong. So better just remove it.

if i dont refresh the page i miss comments. However i have updated the statement